src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -11,6 +11,7 @@ import {
   ModifyAnonymousChatData
 } from '../../../types/repo/AnonymousMessageRepo/InitAnonymousChatData';
 import {getAnonymousChatName} from '../../utils/string/StringUtils';
+import {GROUP_INFO} from '../../hooks/core/constant';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement `import {GROUP_INFO} from '../../hooks/core/constant';` has been added. Ensure that the `GROUP_INFO` constant is correctly defined and exported in the specified path.

```diff
+ import {GROUP_INFO} from '../../hooks/core/constant';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->